android/src/main/java/com/clevertap/react/CleverTapModule.java

```diff
@@ -837,6 +839,21 @@ public void profileGetCleverTapID(Callback callback) {
         callbackWithErrorAndResult(callback, error, result);
     }
 
+    @ReactMethod
+    public void getCleverTapID(final Callback callback) {
+        final CleverTapAPI clevertap = getCleverTapAPI();
+        if (clevertap != null) {
+            clevertap.getCleverTapID(new OnInitCleverTapIDListener() {
+                @Override
+                public void onInitCleverTapID(final String cleverTapID) {
+                    // Callback on main thread
+                    callbackWithErrorAndResult(callback, null, cleverTapID);
+                }
+
+            });
+        }
```

Comment:
@piyush-kukadiya if `clevertap` is null on line 845 then give callback with some proper error

Reply:
Done